source/tutorials/openfaas.rst

```diff
@@ -0,0 +1,363 @@
+.. _openfaas:
+
+OpenFaaS\*
+##########
+
+This tutorial shows how to set up OpenFaaS running on top of a Kubernetes\*
+cluster on |CL|, obtain |CL| based OpenFaaS templates, and develop an example
+function.
+
+.. contents:: :local:
+    :depth: 1
+
+Background
+**********
+
+Functions as a Service (FaaS) is a framework for building serverless functions
+that are ephemeral, automatically scalable, and focused pieces of code running
+within containers to allow developers to focus on application code rather than
+infrastructure nuances.
+
+Many cloud service providers have ready-to-use FaaS offerings which offer a
+high degree of convenience for developers and granular billing based on
+per-second usage. 
+
+If you want an on-premise or self-hosted serverless capability to avoid vendor
+lock-in or simply want more development, OpenFaaS is currently the most
+popular solution in the space based on the number of Github stars on the
+project.
+
+Prerequisites
+*************
+
+For simplicity, this tutorial assumes you have a Kubernetes single node
+cluster with only master node running |CL|.
+
+- For detailed instructions on how to install |CL|, see the :ref:`getting
+  started <get-started>` section.
+
+- For a detailed guide on how to set up Kubernetes, see the documentation on
```

Comment:
The document link seems got lost. I couldn't open it.

Reply:
This is rst syntax to refer to other local docs. It'll point to https://docs.01.org/clearlinux/latest/tutorials/kubernetes.html when it is built locally.

Reply:
@puneetse. Is the dependency on the k8s tutorial, in its current state, a liability or concern?

Reply:
@mvincerx I do not want to block this on the K8s improvements.